geocoding_ban/plugin.json

```diff
@@ -0,0 +1,29 @@
+// This file is the descriptor for the DSS Plugin geocoding_BAN
+{
+    // The identifier of the plugin.
+    // This must be globally unique, and only contain A-Za-z0-9_-
+    "id" : "geocoding_ban",
```

Comment:
DSS plugins generally have names with dashes rather than underscore, so you might want to name that geocoding-ban